osbuild.spec

```diff
@@ -129,7 +130,7 @@ make man
 make -f /usr/share/selinux/devel/Makefile osbuild.pp
 bzip2 -9 osbuild.pp
 
-%pre
+%pre selinux
 %selinux_relabel_pre -s %{selinuxtype}
```

Comment:
Oops, this used to create backups in the `osbuild` package, rather than `osbuild-selinux`. The `*_pre` macro is only for backups, IIRC, so probably not worth any backports, though.

Reply:
Ups, probably copy and paste error from old versions of the guide: https://fedoraproject.org/w/index.php?title=SELinux/IndependentPolicy&oldid=565189#The_%prep_and_%install_Section